src/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 configure_file(version.h.in version.h)
 
+# Add compiler warnings
+if (MSVC)
+	add_compile_options(/W4)
+else()
+	add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
```

Comment:
I thought these are included in `Debug` flags. If not, shouldn't these be added to the CI configuration (`CMakePresetsConfig-CI.json`) rather than to everyone's?